pytest.ini

```diff
@@ -0,0 +1,3 @@
+[pytest]
+filterwarnings =
+    ignore:the imp module is deprecated:DeprecationWarning
```

Comment:
This is a global config and easy to miss in the future. Is this worth explicitly marking tests with `@pytest.mark.filterwarnings`? (see https://docs.pytest.org/en/latest/warnings.html#pytest-mark-filterwarnings)

This could even be set up as a pytest fixture so it looks fairly clean while being explicit.